data/common.yaml

```diff
@@ -8,6 +8,9 @@ lookup_options:
   mlocate::prunepaths:
     merge: unique
 
+mlocate::package_names:
+  - mlocate
```

Comment:
instead of declaring it here, could you move the data to the init.pp?

Reply:
Can do